src/Controller/Action/AddProductVariantToWishlistAction.php

```diff
@@ -104,6 +106,20 @@ public function __invoke(Request $request): Response
         return $response;
     }
 
+    private function addProductToWishlist(
+        WishlistInterface $wishlist,
+        ProductVariantInterface $variant,
+        WishlistProductInterface $wishlistProduct
+    ): void {
+        if ($wishlist->hasProductVariant($variant)) {
+            $message = sprintf('%s variant is already in wishlist.', $wishlistProduct->getProduct()->getName());
+            $this->flashBag->add('error', $this->translator->trans($message));
+        } else {
```

Comment:
Else is not necessary here: https://dev.to/tadeubdev/object-calisthenics-with-php-en-2h4p